pocs/observatory.py

```diff
@@ -218,6 +220,9 @@ def power_down(self):
         """Power down the observatory. Currently does nothing
         """
         self.logger.debug("Shutting down observatory")
+        if self.mount is None:
+            self.logger.info(f'Mount not present, cannot shut down.')
+            return
```

Comment:
We don't want this to return, or at least not before things like the dome. If we lost the mount for some reason we definitely still want the dome to close and shut down properly.  Let's just give a warning but continue with the rest of the shutdown.
```suggestion
```